examples/encryption-provider-config.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiserver.config.k8s.io/v1
 kind: EncryptionConfiguration
 resources:
   - resources:
-      - secrets
+      - "*.*"
```

Comment:
Is there a reason our example encryption includes all resources now? There's usually a performance overhead to encrypting all resources